api/src/main/java/org/pmiops/workbench/impersonation/ImpersonatedFirecloudService.java

```diff
@@ -0,0 +1,19 @@
+package org.pmiops.workbench.impersonation;
+
+import java.io.IOException;
+import javax.annotation.Nonnull;
+import org.pmiops.workbench.db.model.DbUser;
+
+/**
+ * An impersonation-enabled version of {@link org.pmiops.workbench.firecloud.FireCloudService}
+ *
+ * <p>REMINDER: With great power comes great responsibility. Impersonation should not be used in
+ * production, except where absolutely necessary.
+ */
+public interface ImpersonatedFirecloudService {
```

Comment:
here is one idea for isolating more dangerous methods - a separate service which parallels the safer version.  what do you think?